runtime/oti/JITInterface.hpp

```diff
@@ -177,6 +186,8 @@ class VM_JITInterface
 			/* Virtual - unsigned offset is in the low 12 bits, assume the sign bit is set (i.e. the offset is always negative) */
 			jitVTableIndex = 0 - (instruction & 0x00000FFF);
 		}
+#elif defined(J9VM_ARCH_AARCH64)
+		/* To be filled */
```

Comment:
Can these assert / #error for the time being for the same reasons as above?

Reply:
I am going to add assert here, too.